src/app/file-browser/components/file-list-controls/file-list-controls.component.ts

```diff
@@ -339,8 +361,15 @@ export class FileListControlsComponent implements OnInit, OnDestroy, HasSubscrip
       return;
     }
 
-    if (this.selectedItems.length === 1 && this.selectedItems[0] instanceof RecordVO) {
-      this.data.downloadFile(this.selectedItems[0]);
+    if(this.selectedItems.length === 1 && this.selectedItems[0] instanceof RecordVO){
+      if(!this.displayDownloadDropdown){
+       this.displayDownloadOptions()
+      }
+      else{
+        this.displayDownloadDropdown = false
+        this.downloadOptions = []
+      }
+
```

Comment:
I would think we'd need some extra spaces here between parens and brackets - did you run prettier on this file?